docs/src/dev-guide/coding-guidelines.md

```
@@ -53,6 +53,110 @@ To avoid including a state variable in a React Hook's dependency array, you can
+* If the file is too large and needs to be sliced, set the folder name to `xxxStore` and the store
+creation file name to `index.ts`.
+
+## Creation
+
+### Values and actions
+
+Zustand stores two types of data: state variables, i.e. values, and actions, which are functions that
+modify these state variables.
+
+We split values and actions in two different interfaces during the definition: `XxxValues` for values and `XxxActions`
+for actions, both in PascalCase. Then we combine them with a type called `XxxState` for store creation. This is 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)